test_vec/vdaf/Prio3Sum_2.json

```diff
@@ -0,0 +1,230 @@
+{
+    "agg_param": "",
+    "agg_result": 1521,
+    "agg_shares": [
+        "3d90b1d19acf1386",
+        "b5754e2e6430ec79"
+    ],
+    "ctx": "736f6d65206170706c69636174696f6e",
+    "max_measurement": 22,
```

Comment:
This looks wrong, something fishy is happening.

Reply:
Ah, `test_vec_set_type_param()` on `Sum` is just grabbing the wrong field to store in the test vector.

Reply:
GREAT GRAND (will fix0

Reply:
Done, nice catch.